session/txn.go

```diff
@@ -152,12 +144,6 @@ type dirtyTableOperation struct {
 
 // Commit overrides the Transaction interface.
 func (st *TxnState) Commit(ctx context.Context) error {
```

Comment:
Should we remove it now?

Reply:
It's better keep the check to catch bugs.